spec/factories/user_factory.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :user do
+    username "felix_pip"
+    email "felix@pip.com"
+  end
+end
```

Comment:
We prefer to use the single file for factories, because it makes it easier to track a factory down and see what is being set on which object.